README.md

```diff
@@ -20,11 +20,14 @@ limitations under the License.
 [![Build Status][ti]][tl] [![Maven Central][mi]][ml] [![Javadoc][ji]][jl] [![Apache License][li]][ll]
 
 [Apache Accumulo][accumulo] is a sorted, distributed key/value store that provides robust,
-scalable data storage and retrieval. With Apache Accumulo, users can store and manage large
-data sets across a cluster. Accumulo uses [Apache Hadoop]'s HDFS to store its data and
+scalable data storage and retrieval. With Apache Accumulo, users can store and manage large data sets across a cluster. Accumulo uses [Apache Hadoop]'s HDFS to store its data and
```

Comment:
Since this section doesn't change any wording, we should keep the existing 2 lines, rather than merge them into 1.